code/modules/clothing/spacesuits/rig/rig.dm

```diff
@@ -538,6 +534,30 @@
 		wearer.update_inv_back()
 	return
 
+//this makes your modules' overlays be drawn upon your rig's chest if it exists
+/obj/item/weapon/rig/proc/add_item_overlay(obj/item/I)
+	if(istype(I, chest))//e.g. we have parts which are supposed to be on our helmet, so we check
```

Comment:
Why does this check only chest and then stop?

Aren't there modules which add overlays to the helmet, as the comment indicates? Those wouldn't work with this code

Reply:
Okay i got it.
It is better now to left this part. The reason of it - each `rig_module` only has `var/obj/item/weapon/rig/holder` as a reference to it's "owner", it isn't attached to a specific part - a hand or a helmet, so i can't fugire out how to correlate rig_module (e.g. flashlight) to rig's helmet.
Well, no, i can always add some little code to EACH rig_module to ONLY know where it should be drawn and make it more complicated, but this is just a liiitle unnecessarry feature which makes me to do extra changes.

Reply:
it would be a good feature, please do it

a spike coming from the wrist should be an overlay on the gloves, not on the suit